hw/ip/otbn/dv/otbnsim/sim/insn.py

```diff
@@ -181,15 +197,27 @@ def __init__(self, raw: int, op_vals: Dict[str, int]):
         self.offset = op_vals['offset']
         self.grs1 = op_vals['grs1']
 
-    def execute(self, state: OTBNState) -> None:
+    def execute(self, state: OTBNState) -> Optional[Iterator[None]]:
+        # LW executes over two cycles. On the first cycle (where 'cont' is
+        # None), we read the base address, compute the load address and check
+        # it for correctness, then perform the load itself, returning the
+        # result.
+        #
+        # On the second cycle, the 'cont' value is the data we loaded.
```

Comment:
This comment needs updating.

Reply:
Gah! I thought I'd caught them all :-)

Reply:
Damn! It seems that searching for "cont" and then a space was defeated by pesky apostrophes :-)